x-pack/plugins/security_solution/cypress/integration/hosts/session_view.spec.ts

```diff
@@ -132,5 +134,47 @@ describe('Session view', () => {
       cy.get(ALERT_NODE_TEST_ID).first().click();
       cy.location('pathname').should('contain', `app/security/rules/id/${ALERT_RULE_ID}`);
     });
+
+    it('renders child processes correctly', () => {
+      openSessionView(TEST_EVENT_ID);
+
+      // Amount of visible commands on the session view should increase when user clicks on the Child Process dropdown button
+      cy.get(SESSION_COMMANDS).children().its('length').then(lengthBefore =>{
+        const beforeClick = lengthBefore;
+        cy.contains('Child processes').click()
+        cy.contains(FIRST_CHILD_COMMAND).should('exist');
+        cy.get(SESSION_COMMANDS).children().its('length').then(lengthAfter =>{
+          const afterClick = lengthAfter;
+          expect(afterClick).to.be.greaterThan(beforeClick)
+        })
+      })
+
+      //Checks the left margin value for each command line, left margin of child would be more to the right compared the parent 
+      
+      //Get the margin-left value for parent command
+      cy.get(SESSION_COMMANDS).eq(1).then(element => {
+        const win = element[0].ownerDocument!.defaultView
+        const before = win!.getComputedStyle(element[0],'before')
+        const contentValue = before.getPropertyValue('margin-left')
+        const parentCommandLeftMargin = parseInt(contentValue.replace('px',''))
+      //Get the margin-left value for child command and compares both of them
+        cy.get(SESSION_COMMANDS).eq(2).then(element => {
+          const win = element[0].ownerDocument!.defaultView
+          const before = win!.getComputedStyle(element[0],'before')
+          const contentValue = before.getPropertyValue('margin-left')
+          const childCommandLeftMargin = parseInt(contentValue.replace('px',''))
+          expect(parentCommandLeftMargin).to.be.greaterThan(childCommandLeftMargin)
+        })
+      })
+    })
+    
+    it('root escalation', () => {
+      openSessionView(TEST_EVENT_ID);
+
+      // Get the parent div for Root Escalation button and checks if Sudo su and Root Escalation button is with same parent or not
+      cy.contains('Root escalation').parent().contains('sudo').should('exist');
```

Comment:
Right now it only checks the first root escalation. Might want to check if every root escalation corresponds to a `sudo`